llvm/tools/llc/llc.cpp

```diff
@@ -410,6 +429,21 @@ int main(int argc, char **argv) {
 
   if (RemarksFile)
     RemarksFile->keep();
+  if (StringRef(OutputFilename).ends_with(".spv")) {
```

Comment:
If llc generates a '.spv' file, we try to validate it if spirv-val is present. Any issue is reported as warning to the user and the compilation proceeds further.

Thanks